x-pack/plugins/reporting/server/routes/lib/get_document_payload.ts

```diff
@@ -18,42 +21,44 @@ function getDocumentPayloadFn(server) {
     }
   }
 
-  function getCompleted(output, jobType, title) {
-    const exportType = exportTypesRegistry.get(item => item.jobType === jobType);
+  function getCompleted(output: any, jobType: string, title: any) {
+    const exportType = exportTypesRegistry.get((item: any) => item.jobType === jobType);
+    const filename = `${title || 'report'}.${exportType.jobContentExtension}`;
```

Comment:
I feel like `'report'` would stand out in the way it needs to better if it was a const. Something like:
```
const DEFAULT_TITLE = 'report'
```

And, maybe more nitpicky, but it could be good to have a `getFilename(title, extension)` helper function that handles the defaulting of the title